docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss

```diff
@@ -182,6 +188,34 @@
     &:hover:not(.is-active) {
       @include hb-global-color('color', 'black');
 
+      .hb-dark-pattern & {
+        @include hb-global-color('color', 'black');
+
+        & + .hb-main-nav__button,
+        &.hb-nested-toggler::after {
+          @include hb-icon-background('plus', 'black');
```

Comment:
Wow!! There was soooo much to target to make this display as expected! Great job hunting down all the things! 

![giphy-downsized](https://user-images.githubusercontent.com/12678977/96192612-5da83780-0f14-11eb-9ab6-f521e6763e16.gif)